### PR TITLE
[ECS] Updating journald to ECS 8.10 for new `process.thread.capabilities.effective`

### PR DESCRIPTION
**Pull request by kgeller**
## Proposed commit message

Update journald to include new `process.thread.capabilities.effective` ECS field. 

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Relates https://github.com/elastic/integrations/issues/7480
- Requires https://github.com/elastic/beats/pull/36470

### CODE REVIEW

File: packages/journald/docs/README.md [hunk 1]
```diff
@@ -11,31 +11,31 @@ An example event looks as follows:
...
+        "ephemeral_id": "f7858fe6-ce04-46d6-83c3-f45a4e019395",
+        "id": "26693255-8a33-48c9-87cc-3d5f846c4bcd",
         "name": "docker-fleet-agent",
         "type": "filebeat",
-        "version": "8.3.2"
+        "version": "8.11.0"
     },
     "data_stream": {
-        "dataset": "journald.log",
+        "dataset": "journald.logs",
```
Comment (kgeller) [inline packages/journald/docs/README.md:21]: Updated per the sample_event

File: packages/journald/manifest.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-version: 1.0.1
+version: 1.1.0
 description: Collect logs from journald with Elastic Agent.
 type: input
 categories:
   - observability
   - custom
 conditions:
-  kibana.version: "^8.8.0"
+  kibana.version: "^8.11.0"
```
Comment (kgeller) [inline packages/journald/manifest.yml:11]: I assume due to this, we will want to hold merging this PR until 8.11 is released?

Comment (andrewkroh) [reply packages/journald/manifest.yml:11]: While it is true that the new field is only available in 8.11.0 and newer, I don't think we need increase the required version. Everyone about the package will continue to work with older versions.

File: packages/journald/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.0"
+  changes:
+    - description: Upgrade journald to ECS 8.10.
```
Comment (andrewkroh) [inline packages/journald/changelog.yml:4]: Consider adding a second entry to specifically mention that `process.thread.capabilities.effective` was added to the mapping for Elastic Agent 8.11 and newer.

**Review by andrewkroh — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]
I left some comments, but I don't need to re-review.

**Review by cmacknz — APPROVED** [review]

**Pull request merged**